src/controller/FileBrowserController.vala

```diff
@@ -86,8 +86,8 @@ namespace BrickManager {
                                 "--",
                                 "/bin/bash",
                                 "-c",
-                                "/usr/bin/tty >%s;tty=$(/bin/cat %s);/bin/chown %s: \$tty;/sbin/runuser --login --shell /bin/bash --command '(cd %s && exec %s)' %s"
-                                    .printf (ttyname_path, ttyname_path, USER_NAME, file.get_parent().get_path (), file.get_path (), USER_NAME),
+                                "/usr/bin/tty >%s;tty=$(/bin/cat %s);/bin/chown %s: \$tty;/usr/bin/sudo -u %s /bin/bash -c '(cd %s && exec %s)'"
```

Comment:
I would like to keep long option names here. Also, I would like to keep `--login` so that the environment is the same as if someone logged in remotely as `robot` (e.g. `~/.bashrc` is run).

And, it shouldn't be needed, but we could also add `--non-interactive` to be extra safe.